aat/src/aat/resources/features/F-094 NoC ChallengeQuestions/S-094.1.td.json

```diff
@@ -36,7 +36,8 @@
 					"display_context_parameter": "#DATETIMEENTRY(dd-MM-yyyy)",
 					"challenge_question_id": "NoCChallenge",
 					"answer_field": "${[scenarioContext][customValues][no_dynamic_injection_${OrganisationField.OrganisationName}|${[scenarioContext][customValues][no_dynamic_injection_${OrganisationField.OrganisationID}:[Claimant],${[scenarioContext][customValues][no_dynamic_injection_${OrganisationField.OrganisationName}|${[scenarioContext][customValues][no_dynamic_injection_${OrganisationField.OrganisationID}:[Defendant]",
-					"question_id": "NoCChallengeQ1"
+					"question_id": "NoCChallengeQ1",
```

Comment:
I think it's worth adding an example where it's `true` here as well - to prove the issue raised in my other comment isn't an issue anymore (after those changes have been made).

Reply:
updated to validate the true condition